config.yaml

```diff
@@ -2,18 +2,22 @@
 # See LICENSE file for licensing details.
 
 options:
-  git-sync::repo:
+  git_sync_repo:
     type: string
     description: URL to repo to sync against
-  git-sync::branch:
+  git_sync_branch:
     type: string
     description: branch to sync
     default: main
-  git-sync::wait:
+  git_sync_wait:
     type: int
     description: git-sync wait interval between syncs (seconds).
     default: 60
-  prometheus_relpath:
+  prom_rules:
```

Comment:
`prometheus_alert_rules_path` please. Without the "path" someone will try something dumb and add the content of a file

Reply:
also, we're not in a shortage of letters, so we might a s well be as descriptive as possible 😅

Reply:
Done.